### PR TITLE
[catalyst-africa] Standarize profileList

### PR DESCRIPTION
**Pull request by AIDEA775**
Continuation of #3721 
Ref: #3631 

I had to add:
```yaml
node_selector:
  node.kubernetes.io/instance-type: r5.xlarge
```
because AWS launches an `r5.16xlarge` instance by default.

I made some modifications to catalyst-latam as well.

### CODE REVIEW

**Review by consideRatio — COMMENTED** [review]
### Available CPU

In this example, cpu advertised is 1/10 of a CPU - in practice I'm quite confident it will be closer to 1 full CPU at all times since usage is typically very low.

If users think they only get 10% of a CPU, I figure they may want to use bigger machines to get at least 1 CPU for example. If that happens, the cloud cost increase etc.

```yaml
                display_name: "Small: 750MB RAM and 1/10 CPU"
                kubespawner_override:
                  mem_guarantee: 732421K
                  mem_limit: 2G
                  cpu_guarantee: 0.093
                  cpu_limit: 1
```

In the Medium example I saw that the cpu guarantee wasn't what was featured - but a midway between cpu guarantee and limit, making the labelling practice incosnsistent.

I think its complicated to communicate about this, but I'd say that "up to <cpu limit>" is a good choice on what to label, and ignore mentioning the lower limit as its often not very relevant.

```yaml
              mem_3_0:
                display_name: "Medium: 3GB RAM and 1/2 CPU"
                kubespawner_override:
                  mem_guarantee: 2929687.5K
                  mem_limit: 6G
                  cpu_guarantee: 0.375
                  cpu_limit: 2
```

### On relative label names (Small / Medium / Large etc)

I'm conflicted about the use of such labels with reasons like:

- Relative labels are easy to speak about - this can be helpful!
- Relative labels are tricky to maintain:
  - Users will refer to these options among themselves and in contact with 2i2c, but 2i2c will need to check what they really mean often once they do
  - Relative labels are hub specific and may not align with other hubs, and some users could perhaps show up to use different hubs
  - Relative labels may also change over time in a given hub, possibly breaking an expectation over time
  - Relative labels are relative also to each other - what would we call a new option sized between Small and Medium?
  - The absolute ram/cpu info will remain relevant for users, making use of relative labels not necessarily make this information omittable anyhow.

**Review by consideRatio — APPROVED** [review]
Thank you for working this @AIDEA775!!!

Marking this as approved right away as I saw no critical blocker, I think its good if you are able to iterate quickly on this!

File: config/clusters/catalystproject-latam/common.values.yaml [hunk 1]
```diff
@@ -18,50 +18,28 @@ jupyterhub:
         slug: jupyter
         default: true
         kubespawner_override:
-          image: jupyter/scipy-notebook:2023-06-27
+          image: jupyter/scipy-notebook:2023-10-20
```
[the remark this answers is not in the document]
Comment (AIDEA775) [reply config/clusters/catalystproject-latam/common.values.yaml]: Ahh! It was strange to me that there are no newer images available. Thanks you!

**Comment by AIDEA775** [comment]
I agree with displaying limits instead of guarantees. I have some issues with how JupyterLab shows memory in the status bar, I will work on that in another PR.

**Pull request merged**